XSLT/memphisp16108coll5dctomods.xsl

```diff
@@ -0,0 +1,227 @@
+    </xsl:template>
+    
+    <!-- title -->
+    <xsl:template match="dc:title">
+        <title><xsl:apply-templates/></title>
+    </xsl:template>
+    
+    <!-- abstract -->
+    <xsl:template match="dc:description">
+        <abstract><xsl:apply-templates/></abstract>
+    </xsl:template>
+    
+    <!-- Creators & Contributors -->
+    <xsl:template match="dc:creator">
+        <name>
+            <namePart><xsl:apply-templates/></namePart>
+            <role>
+                <roleTerm type="text" valueURI="http://id.loc.gov/vocabulary/relators/cre">Creator</roleTerm>
+            </role>
+        </name>
+    </xsl:template>
+    <xsl:template match="dc:contributor">
+        <xsl:choose>
+            <xsl:when test="starts-with(., 'Gift of')">
+                <xsl:variable name="contributor-only" select="replace(., 'Gift of ', '')"/>
+                <name>
+                    <namePart><xsl:value-of select="$contributor-only"/></namePart>
+                    <role>
+                        <roleTerm type="text" valueURI="http://id.loc.gov/vocabulary/relators/ctb">Contributor</roleTerm>
+                    </role>
+                </name>    
+            </xsl:when>
+        </xsl:choose>
+        <xsl:choose>
+            <xsl:when test="starts-with(., 'Donated by')">
+                <xsl:variable name="contributor-only-2" select="replace(., 'Donated by ', '')"/>
+                <name>
+                    <namePart><xsl:value-of select="$contributor-only-2"/></namePart>
+                    <role>
+                        <roleTerm type="text" valueURI="http://id.loc.gov/vocabulary/relators/ctb">Contributor</roleTerm>
+                    </role>
+                </name>    
+            </xsl:when>
+            <xsl:otherwise>
+                <name>
+                    <namePart><xsl:apply-templates/></namePart>
+                    <role>
+                        <roleTerm type="text" valueURI="http://id.loc.gov/vocabulary/relators/ctb">Contributor</roleTerm>
+                    </role>
+                </name>
+            </xsl:otherwise>
+        </xsl:choose>
+    </xsl:template>
+    
+    <!-- form -->
+    <xsl:template match="dc:format">
+        <physicalDescription><form><xsl:apply-templates/></form></physicalDescription>
+    </xsl:template>
+    
+    <!-- originInfo> -->
+    <xsl:template match="dc:date">
+        <originInfo><dateCreated><xsl:apply-templates/></dateCreated></originInfo>
+    </xsl:template>
+    
+    <!-- subject(s) -->
+    <!-- for subjects, whether they contains a ';' or not -->
+    <xsl:template match="dc:subject">
+        <xsl:variable name="subj-tokens" select="tokenize(., '; ')"/>
+        <xsl:for-each select="$subj-tokens">
+            <xsl:choose>
+                 <xsl:when test="ends-with(., ';')">
+                    <subject>
+                        <topic>
+                            <xsl:apply-templates select="substring(., 1, string-length(.) -1)"/>
```

Comment:
@mlhale7 We're using the wrong xsl function here.  Like on 184, we need to us value-of:

```
<xsl:value-of select="substring(., 1, string-length(.) -1)"/>
```

Reply:
Thanks for catching this!

Reply:
Looks good.